File: Boundary Count/JAVA/encoder.java
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class encoder {
    private static Map<Character, String> MORSE_CODE_MAP = new HashMap<Character, String>() {{
        put('A', ".-"); put('B', "-..."); put('C', "-.-."); put('D', "-.."); put('E', ".");
        put('F', "..-."); put('G', "--."); put('H', "...."); put('I', ".."); put('J', ".---");
        put('K', "-.-"); put('L', ".-.."); put('M', "--"); put('N', "-."); put('O', "---");
        put('P', ".--."); put('Q', "--.-"); put('R', ".-."); put('S', "..."); put('T', "-");
        put('U', "..-"); put('V', "...-"); put('W', ".--"); put('X', "-..-"); put('Y', "-.--");
        put('Z', "--..");
    }};

    public static String encode(String message) {
        String encoded_message = "";
        for (char ch : message.toCharArray()) {
            if (ch == ' ') {
                continue;
            }
            char upper_ch = Character.toUpperCase(ch);
            if (MORSE_CODE_MAP.containsKey(upper_ch)) {
                String morse = MORSE_CODE_MAP.get(upper_ch);
                int ascii_val = upper_ch;
                encoded_message += morse + upper_ch + String.valueOf(ascii_val);
            }
        }
        return encoded_message;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a word to encode: ");
        String user_input = scanner.nextLine();
        String encoded_result = encode(user_input);
        System.out.println("Encoded result: " + encoded_result);
        scanner.close();
    }
}

File: Boundary Count/PYTHON/encoder.py
MORSE_CODE_MAP = {
    'A': ".-", 'B': "-...", 'C': "-.-.", 'D': "-..", 'E': ".",
    'F': "..-.", 'G': "--.", 'H': "....", 'I': "..", 'J': ".---",
    'K': "-.-", 'L': ".-..", 'M': "--", 'N': "-.", 'O': "---",
    'P': ".--.", 'Q': "--.-", 'R': ".-.", 'S': "...", 'T': "-",
    'U': "..-", 'V': "...-", 'W': ".--", 'X': "-..-", 'Y': "-.--",
    'Z': "--.."
}

def encode(message):
    encoded_message = ""
    for ch in message:
        if ch == ' ':
            continue
        upper_ch = ch.upper()
        if upper_ch in MORSE_CODE_MAP:
            morse = MORSE_CODE_MAP[upper_ch]
            ascii_val = ord(upper_ch)
            encoded_message += morse + upper_ch + str(ascii_val)
    return encoded_message

if __name__ == "__main__":
    user_input = input("Enter a word to encode: ")
    encoded_result = encode(user_input)
    print("Encoded result:", encoded_result)